gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -186,6 +192,151 @@ def get_hgdp_tgp_v4_exome_duplicates(
+            "population": populations.population,
+            "latitude": populations.latitude,
+            "longitude": populations.longitude,
+            "gnomad_labeled_subpop": populations.population.lower(),
+        },
+        "high_quality": ~hard_filtered & ~outlier,
+    }
+    updated_ht = update_structured_annotations(
+        ht,
+        sample_annotations_to_update,
+        annotation_update_label="sample_annotations_updated",
+    )
+    updated_ht = updated_ht.annotate()
+
+    updated_ht = updated_ht.checkpoint(new_temp_file("hgdp_tgp_meta_update", "ht"))
+    updated_counts = updated_ht.aggregate(
+        hl.struct(
+            n_hard_filtered=hl.agg.count_where(
+                updated_ht.gnomad_sample_filters.hard_filtered
+            ),
+            n_related_subset=hl.agg.count_where(
+                updated_ht.relatedness_inference.related
+            ),
+            n_related_nonsubset=hl.agg.count_where(
+                ~updated_ht.relatedness_inference.related
+                & updated_ht.gnomad_sample_filters.related_to_nonsubset
+            ),
+            n_duplicate_to_exomes=hl.agg.count_where(
+                ~updated_ht.relatedness_inference.related
+                & ~updated_ht.gnomad_sample_filters.related_to_nonsubset
+                & updated_ht.gnomad_sample_filters.v4_exome_duplicate
+            ),
+            n_outlier=hl.agg.count_where(
+                updated_ht.hgdp_tgp_meta.subcontinental_pca.outlier
+            ),
+            n_release=hl.agg.count_where(updated_ht.gnomad_release),
+            n_diff=hl.agg.explode(
+                lambda x: hl.agg.counter(x), updated_ht.sample_annotations_updated
+            ),
+        )
+    )
+    logger.info(
+        """%d sample(s) hard filtered (%d more samples compared to v3.1.2 subset release);
+        %d samples are pop outliers (%d samples different compared to v3.1.2 subset release);
+        %d samples have different population labels compared to v3.1.2 subset release;
+        %d samples related within the subset (%d samples different compared to v3.1.2 subset release);
+        %d samples further filtered out due to their relatedness to samples outside the subset;
+        %d samples filtered out because they are duplicated in the v4.0 exomes release;
+        %d samples will be in the v4.0 release, compared to 3280 in the v3.1.2 release.""",
+        updated_counts["n_hard_filtered"],
+        updated_counts["n_diff"]["gnomad_sample_filters.hard_filtered"],
+        updated_counts["n_outlier"],
+        updated_counts["n_diff"]["hgdp_tgp_meta.subcontinental_pca.outlier"],
+        updated_counts["n_diff"]["hgdp_tgp_meta.population"],
+        updated_counts["n_related_subset"],
+        updated_counts["n_diff"]["relatedness_inference.related"],
+        updated_counts["n_related_nonsubset"],
+        updated_counts["n_duplicate_to_exomes"],
+        updated_counts["n_release"],
+    )
+    return updated_ht
```

Comment:
When I tested, this chunck of logger is not working at all. 
https://console.cloud.google.com/dataproc/jobs/9ded3f68943149a8a0de33c9cae7e547/monitoring?region=us-central1&project=broad-mpg-gnomad
Do you know why?

Reply:
I think there is something wrong with the loggers all of a sudden. Mike mentioned an issue with the loggers in the release code too. I will look into it, but I don't think it needs to hold up merging